client/src/components/pages/application/Accounts/index.js

```diff
@@ -0,0 +1,184 @@
+  useEffect(() => {
+    getUserID().then((data) => {
+      if (data.message === 'you are authorized') {
+        setId(data.userId);
+      }
+    });
+    if (UserId) {
+      axios.get(`/api/v1/applicants/${UserId}`).then(({ data: { user } }) => {
+        setGitHub(user.githubLink);
+        setFreeCode(user.freeCodeCampLink);
+        setCodeWars(user.codeWarsLink);
+        setName(user.fullName);
+      });
+    }
+  }, [UserId]);
+  const Next = () => {
+    AccountsVallation({ githubLink, freeCodeCampLink, codeWarsLink })
+      .then(() =>
+        axios.patch(`/api/v1/applicants/${UserId}`, {
+          githubLink,
+          freeCodeCampLink,
+          codeWarsLink,
+          joinDiscord,
+        })
+      )
+      .then(() => history.push('/tasks'))
+      .catch(({ errors }) => throwMessage(errors));
+  };
+
+  const Name = userName.split(' ')[0];
+  return (
+    <div className="Container_page">
+      <Helmet>
+        <title>Accounts</title>
+      </Helmet>
+      <img src={backGround} alt="backGround" className="backGround" />
+      {UserId && userName ? (
+        <>
+          <div className="text_Welcome">
+            <Typography variant="h3" color="default">
+              Welcome, {Name}
+            </Typography>
+          </div>
+          <div className="Form_container">
+            <Typography variant="h5">Accounts</Typography>
+            <>
+              <div className="label_container">
+                <img src={gitHub} alt="GitHub" />
+                <Typography className="label">Github</Typography>
+              </div>
+              <TextField
+                name="githubLink"
+                value={githubLink}
+                onChange={(e) => setGitHub(e.target.value)}
+                isError={
+                  message.includes('Enter GitHub link') ||
+                  message.includes('Error Github link')
+                }
+                message={
+                  message.includes('Error Github link')
+                    ? 'Error Github link'
+                    : ''
+                }
+                placeholder="Enter Your Github Profile"
+              />
+            </>
+            <>
+              <div className="label_container">
+                <img src={freeCodeCamp} alt="FreeCode Camp" />
+                <Typography className="label">FreeCodeCamp</Typography>
+              </div>
+              <TextField
+                name="freeCodeCampLink"
+                value={freeCodeCampLink}
+                onChange={(e) => setFreeCode(e.target.value)}
+                isError={
+                  message.includes('Enter FreeCodeCamp link') ||
+                  message.includes('Error FreeCodeCamp link')
+                }
+                message={
+                  message.includes('Error FreeCodeCamp link')
+                    ? 'Error FreeCodeCamp link'
+                    : ''
+                }
+                placeholder="Enter Your FreeCode Camp Profile"
```

Comment:
I prefer to add the example link as placeholder and change the value for text field to  empty string. 
```suggestion
                placeholder="ex: https://www.freecodecamp.org/code_academy"
```